streamlit_demo_apps/README.md

```diff
@@ -1,15 +1,19 @@
 # Deploying Streamlit Apps
 
-To deploy Streamlit apps, they need to be written and committed to a public GitHub repository.
+This guide will show you how to quickly spin up a Nillion Testnet connected, live Streamlit app from our template using the simple generate-streamlit-app script. Once your app is ready, you can easily deploy it by pushing it to a public GitHub repo. Just follow these steps to get started:
```

Comment:
> a Nillion Testnet connected, live Streamlit app f

this reads a bit odd - but might be me.

Reply:
How about "Follow the steps to deploy a live Streamlit app for your Nada program. The app will connect to the Nillion Testnet to store your Nada program, store secret inputs (or use computation time secrets), and run blind computation"